datacubes/model-run.schema.json

```diff
@@ -0,0 +1,146 @@
+{
+    "$id": "https://github.com/uncharted-causemos/docs/blob/model-schemas/datacubes/model-run.schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "description": "World Modelers Model Run Metadata Schema",
+    "required": [
+        "id",
+        "model_name",
+        "model_id",
+        "created_at",
+        "data_paths",
+        "parameters",
+        "tags"
+    ],
+    "title": "Model Metadata Schema",
+    "type": "object",
+    "properties": {
+        "id": {
+            "$id": "#/properties/id",
+            "description": "A unique model id",
+            "examples": [
+                "123e4567-e89b-12d3-a456-426614174000"
+            ],
+            "title": "Model ID",
+            "type": "string"
+        },
+        "model_name": {
+            "$id": "#/properties/model_name",
+            "title": "Model Name",
+            "description": "The model name",
+            "type": "string",
+            "examples": [
+                "DSSAT"
+            ]
+        },
+        "model_id": {
+            "$id": "#/properties/model_id",
+            "title": "Model ID",
+            "description": "The ID of the model that this run belongs to",
+            "type": "string",
+            "$ref": "./model.schema.json#/properties/id",
+            "examples": [
+                "123e4567-e89b-12d3-a456-426614174000"
+            ]
+        },
+        "created_at": {
+            "$id": "#/properties/created_at",
+            "title": "Model Registration Time",
```

Comment:
Should this be run creation time, not model registration time?

Reply:
Yup. Thanks!